### PR TITLE
resolve #1

### LINKED ISSUE #1: error when linting: missing typescript-eslint/eslint-plugin
when try to lint files, it shows the following error:

Checking eslinting...

Oops! Something went wrong! :(

ESLint: 6.8.0.

ESLint couldn't find the plugin "@typescript-eslint/eslint-plugin".

(The package "@typescript-eslint/eslint-plugin" was not found when loaded as a Node module from the directory "/home/jliang01/SharedSpace/git/fortinet/ftnt-devops-ci".)

It's likely that the plugin isn't installed correctly. Try reinstalling by running the following:

    npm install @typescript-eslint/eslint-plugin@latest --save-dev

The plugin "@typescript-eslint/eslint-plugin" was referenced from the config file in "--config".

If you still can't figure out the problem, please stop by https://gitter.im/eslint/eslint to chat with the team.

### PR DESCRIPTION
**Pull request by JaydenLiang**
missing typescript eslint plugin

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
         "ftnt-devops-ci": "bin/ftnt-devops-ci.js"
     },
     "dependencies": {
+        "@typescript-eslint/eslint-plugin": "^2.22.0",
```
Comment (JaydenLiang) [inline package.json]:
```suggestion
        "@typescript-eslint/eslint-plugin": "^2.22.0",
```
looks good here.

Comment (Joel-Cripps) [reply package.json]:
```suggestion
        "@typescript-eslint/eslint-plugin": "^2.22.0",
```
Blah blah blah

Comment (Joel-Cripps) [inline package.json]:
```suggestion
        "@typescript-eslint/eslint-plugin": "^2.21.0",
```

File: package.json [hunk 2]
```diff
@@ -28,6 +28,7 @@
         "ftnt-devops-ci": "bin/ftnt-devops-ci.js"
     },
     "dependencies": {
+        "@typescript-eslint/eslint-plugin": "^2.21.0",
```
Comment (JaydenLiang) [inline package.json]:
```suggestion
        "@typescript-eslint/eslint-plugin": "^2.22.0",
```

**Pull request merged**